src/content/docs/en/pages/devtools/cli/azion-cli/commands/create/create.mdx

```diff
@@ -390,7 +390,7 @@ The `--cnames` option is a list of domains' CNAMEs to associate with the domain.
 
 #### digital-certificate-id
 
-The `--digital-certificate-id` option sets the digital certificate's unique identifier. It can be an *integer* or *null*.
+The `--digital-certificate-id` option sets the digital certificate's unique identifier. It can be an *integer*, *null* or the string `lets_encrypt`. If `lets_encrypt` is informed, then a let's encrypt certificate will be created for you.
```

Comment:
```suggestion
The `--digital-certificate-id` option sets the digital certificate's unique identifier. It can be an *integer*, *null* or the string `lets_encrypt`. If `lets_encrypt` is informed, then a Let's Encrypt certificate will be created for you.
```